mbed_app.json

```diff
@@ -21,10 +21,13 @@
             "help": "The password string to use for this APN, set to 0 if none",
             "value": 0
         },
-        "trace-level": {
```

Comment:
I guess that `trace-level` should not be removed?

Reply:
My mistake. added back.